486_Project/Characters_Class.py

```diff
@@ -1,10 +1,12 @@
+#dndbeyond
+
 class Warrior:
     """
     warrior class for player
     """
     def __init__(self):
         # [strength, dexterity, intelligence, constitution]
-        self.base_stats = [7, 5, 5, 7]
+        self.base_stats = [6, 4, 4, 6]
 
         # weapon
         self.weapon = [1, 0, 0, 1, "Ole Trusty Sword"]
```

Comment:
Does the [1,0,0,1] relate to [str, dex, int, con]? I think this is a great way to implement item buffs, if so!